src/Lib/LanguagesLib.php

```diff
@@ -640,6 +641,7 @@ public static function getLanguageDirection($lang) {
             "snd",
             "div",
             "otk",
+            "chg"
```

Comment:
To allow automatic detection of the writing direction, as @soliloquist-tatoeba requested, this should be added to `$autoLangs` below, instead of `$rightToLeftLangs`.

Also, if you add a comma to the end of the line now, you won't have to change the line later when something gets added below. It doesn't make a big difference either way; I just wanted to point it out.

Reply:
Thanks a lot, @Yorwba! I've fixed this

https://github.com/Tatoeba/tatoeba2/pull/2095/commits/1d5e13c5a8edd3a1a093803583e997b928a0f214